configure.ac

```diff
@@ -860,7 +860,7 @@ AC_ARG_ENABLE([production],
               [AS_HELP_STRING([--enable-production],
                               [Determines how to run the compiler.])])
 
-case "X-$enable-production" in
+case "X-$enable_production" in
```

Comment:
The actual value checked should be $enableval or $enable_<feature>, with an underscore, not a hyphen.